src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs

```diff
@@ -68,14 +81,106 @@ public int DefaultBufferSize
         }
 
         /// <summary>
-        /// Determines whether null values of properties are ignored or whether they are written to the JSON.
+        /// Determines whether null values are ignored during serialization and deserialization.
+        /// The default value is false.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">
+        /// Thrown if this property is set after serialization or deserialization has occurred.
+        /// </exception>
+        public bool IgnoreNullValues
+        {
+            get
+            {
+                return _ignoreNullValues;
+            }
+            set
+            {
+                VerifyMutable();
+                _ignoreNullValues = value;
+            }
+        }
+
+        /// <summary>
+        /// Determines whether read-only properties are ignored during serialization and deserialization.
+        /// A property is read-only if it contains a public getter but not a public setter.
```

Comment:
Should this sentence be a remark instead? cc @mairaw, @rpetrusha

Reply:
Yes @ahsonkhan.

Reply:
@steveharter, please move `A property is read-only if it contains a public getter but not a public setter.` to a remark instead.